src/pages/ShoppingList.vue

```diff
@@ -41,7 +41,7 @@
             color="primary"
             size="lg"
             icon="add"
-            to="/addItems"
+            :to="`${$route.params.id}/addItems`"
```

Comment:
This is still currently WIP.
AddItems is still pointing to the single list while it should be a list of current items added by the user

Reply:
The listItems store should follow the new db flattened db structure. ( a list for every users)